source/geod24/concurrency.d

```diff
@@ -1734,38 +1884,155 @@ public class Channel (T)
 
```

Comment:
The method documentation needs to be updated. It doesn't say what closing the channel actually does. And it looks like it notifies all waiting receivers/senders.